contentcuration/contentcuration/frontend/channelEdit/pages/trash/TrashModal.vue

```diff
@@ -0,0 +1,266 @@
+  import BottomToolBar from 'shared/views/BottomToolBar';
+  import ResizableNavigationDrawer from 'shared/views/ResizableNavigationDrawer';
+  import ActionLink from 'edit_channel/sharedComponents/ActionLink';
+  import MessageDialog from 'shared/views/MessageDialog';
+
+  export default {
+    name: 'TrashModal',
+    components: {
+      ContentNodeIcon,
+      BottomToolBar,
+      ResizableNavigationDrawer,
+      ResourcePanel,
+      ActionLink,
+      MessageDialog,
+    },
+    data() {
+      return {
+        loading: false,
+        previewNodeId: null,
+        selected: [],
+        showConfirmationDialog: false,
+      };
+    },
+    computed: {
+      ...mapGetters('currentChannel', ['currentChannel']),
+      headers() {
+        return [
+          {
+            text: this.$tr('selectAllHeader'),
+            align: 'left',
+            sortable: false,
+            selectAll: true,
+            value: 'title',
+          },
+          {
+            text: this.$tr('deletedHeader'),
+            value: 'modified',
+            align: 'right',
+            sortable: false,
+          },
+        ];
+      },
+      sortedItems() {
+        return sortBy(this.items, 'modified').reverse();
+      },
+      items() {
+        return [
+          {
+            id: 'test1',
+            title: 'Item',
+            kind: 'video',
+            modified: new Date(2020, 1, 20),
+          },
+          {
+            id: 'test2',
+            title: 'Item',
+            kind: 'audio',
+            modified: new Date(2020, 2, 1),
+          },
+          {
+            id: 'test3',
+            title: 'Topic',
+            kind: 'topic',
+            modified: new Date(2020, 1, 1),
+          },
+        ];
+      },
+      backLink() {
+        return {
+          name: this.$route.matched[this.$route.matched.length - 2].name,
+          query: this.$route.query,
+          params: this.$route.params,
+        };
+      },
+    },
+    methods: {
+      ...mapActions('contentNode', ['deleteContentNode']),
+      deleteNodes() {
+        let text = this.$tr('deleteSuccessMessage', { count: this.selected.length });
+        this.selected.forEach(id => {
+          this.deleteContentNode(id);
+        });
+        this.showConfirmationDialog = false;
+        this.toggleSelectAll(false);
+        this.$store.dispatch('showSnackbar', { text });
+      },
+      toggleSelectAll(selectAll) {
+        this.selected = selectAll ? this.items.map(i => i.id) : [];
+      },
+      getItemBackground(id) {
+        return this.previewNodeId === id ? this.$vuetify.theme.greyBackground : 'transparent';
+      },
+    },
+    $trs: {
+      trashModalTitle: 'Removed items',
+      trashEmptyText: 'Trash is empty',
+      trashEmptySubtext: 'Content removed from channel will appear here',
+      selectAllHeader: 'Select all',
+      deletedHeader: 'Deleted',
+      itemCountText: '{count, plural,\n =1 {# item}\n other {# items}}',
```

Comment:
I'm not familiar with this syntax. Is this standard ICU?

Reply:
Yes, it shoul match the [guidelines](http://userguide.icu-project.org/formatparse/messages)